charts/intel-gpu-resource-driver/templates/gpu-resource-driver.yaml

```
@@ -0,0 +1,235 @@
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: SYSFS_ROOT
+          value: "/sysfs"
+        # Use this to tell kubelet-plugin where the DRI devices nodes should be.
+        # This will be prefix for CDI devices, runtime will try to mount devices
+        # with this prefix into workloads.
+        #- name: DEV_DRI_PATH
+        #  value: "/fake/dri"
+        volumeMounts:
+        - name: plugins-registry
+          mountPath: /var/lib/kubelet/plugins_registry
+        - name: plugins
+          mountPath: /var/lib/kubelet/plugins
+        - name: cdi
+          mountPath: /etc/cdi
+        - name: varruncdi
+          mountPath: /var/run/cdi
+        - name: defaults
+          mountPath: "/defaults"
+          readOnly: true
+        - name: sysfs
+          mountPath: "/sysfs"
+        securityContext:
+           {{- toYaml .Values.kubeletPlugin.containers.plugin.securityContext | nindent 10 }}
+      volumes:
+      - name: plugins-registry
+        hostPath:
+          path: /var/lib/kubelet/plugins_registry
+      - name: plugins
+        hostPath:
+          path: /var/lib/kubelet/plugins
+      - name: cdi
+        hostPath:
+          path: /etc/cdi
+      - name: varruncdi
+        hostPath:
+          path: /var/run/cdi
+      - name: sysfs
+        hostPath:
+          path: /sys
+      - name: defaults
+        configMap:
+          name: defaults
+      {{- with .Values.kubeletPlugin.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.kubeletPlugin.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.kubeletPlugin.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "intel-gpu-resource-driver.name" . }}-controller
+  namespace: {{ include "intel-gpu-resource-driver.namespace" . }}
+  labels:
+     {{- include "intel-gpu-resource-driver.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "intel-gpu-resource-driver.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.controller.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "intel-gpu-resource-driver.templateLabels" . | nindent 8 }}
+    spec:
+      serviceAccount: intel-gpu-resource-driver-service-account
+      serviceAccountName: {{ include "intel-gpu-resource-driver.serviceAccountName" . }}
+      containers:
+      - name: controller
+        image:  {{ include "intel-gpu-resource-driver.fullimage" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/controller", "-v", "6"]
```

```suggestion
        command: ["/controller"]
```
Let's not enable such verbosity by default.